### PR TITLE
Add experiences archetype

### LINKED ISSUE #51: Missing archetype for experiences widget
**Describe the bug**

There is a missing archetype for experiences widget. I'll fix this when I can. Just documenting it now.
New widgets should have the archetype made so new widgets can be generated automatically with the archetype template.

**To Reproduce**

1. On root folder of the website with hugo theme installed
2. Enter the following command in terminal

```
hugo new -k widgets/experiences test/test_experiences.md
```
3. The new file does not show the configuration used for experiences

**Expected behavior**

A template for the experiences widget be made with the experiences widget configuration parameters.

**Screenshots**
N/A

**Environment**

 - Hugo Version: [e.g. v0.80.0]
 - Eureka Version: [e.g. v0.5.0]

**Additional context**

I'll add a blank archetype soon and PR it. Just adding this before I forget.

### PR DESCRIPTION
**Pull request by kimlaberinto**
Resolves #51 where the archetype was missing

### CODE REVIEW

File: archetypes/widgets/experiences.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    location:
+    writeup:
+
+  - title:
+    organization:
+      name: 
+      url:
+    dates:
+    location:
+    writeup:
```
Comment (wangchucheng) [inline archetypes/widgets/experiences.md:19]: I think we can just leave one item here and people can add themselves.

**Review by wangchucheng — COMMENTED** [review]
Thanks for the PR! This is very helpful!

File: archetypes/widgets/experiences.md (hunk 1 above)
Comment (kimlaberinto) [reply archetypes/widgets/experiences.md]: Sounds good! Commit added

**Comment by kimlaberinto** [comment]
No problem! You're welcome. I updated it to only one item now.

**Comment by wangchucheng** [comment]
Thanks for your contribution!

**Pull request merged**